core/modules/user/templates/user-simplified-page.tpl.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @file
+ * Theme template for the login forms (for example, "Log in" and "Create new
+ * Account") when shown on simplified (layout suppressed) pages.
+ *
+ * Available variables:
+ *   - $form: The form render array.
+ *   - $links: A render array with related form links.
+ *
+ * @since 1.30.0 Template added
+ * @ingroup themeable
+ */
+?>
+
+<div class="<?php print implode(' ', $classes); ?>">
+  <div class="user-simplified-page-wrapper">
+    <div class="user-simplified-page-wrapper-inner">
+      <?php if ($logo_image): ?>
+        <div class="user-simplified-page-logo">
+          <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home" class="logo">
+            <?php print $logo_image; ?>
+          </a>
+        </div>
+      <?php endif; ?>
+      <h1 class="site-name">
+        <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home"><span><?php print $site_name; ?></span></a>
+      </h1>
```

Comment:
In my testing it looks better to show just the logo *or* the site name, not both. I think this should be the default, though I do expect it will depend on the logo.

```suggestion
      <?php else: ?>
        <h1 class="site-name">
          <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home"><span><?php print $site_name; ?></span></a>
        </h1>
      <?php endif; ?>